emulator.c

```diff
@@ -637,6 +651,12 @@ is_sign_flag_set(i8080 *cpu)
   return (cpu->flags & FLAG_S) != 0;
 }
 
+bool
+is_zero_flag_set(i8080 *cpu)
+{
+  return (cpu->flags & FLAG_Z) == 0;
```

Comment:
Doesn't this return true when the flag is not set?

Reply:
Ah I think I found the issue. I was misusing the update zero flag function in my tests so my tests were backwards.